includeNoKotlin.gradle

```diff
@@ -65,7 +65,7 @@ if (System.getenv("JTRANCS_DEPLOY") != null) {
 			mavenDeployer {
 				beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
 
-				repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
+				repository(url: "https://oss.sonatype.org/content/repositories/snapshots/") {
```

Comment:
Also I don't know how to actual publish automatically from sonatype. I usually do the deploy and then manually release what's deployed. Do you know if it can be done automatically?